src/sentry/api/serializers/models/event.py

```diff
@@ -247,7 +247,7 @@ def serialize(self, obj, attrs, user):
 
         d = {
             'id': six.text_type(obj.id),
-            'groupID': six.text_type(obj.group_id),
+            'groupID': six.text_type(obj.group_id) if obj.group_id else None,
```

Comment:
This change obviously has some pretty wide-ranging implications. Not really sure how to give constructive feedback other than pointing out that hopefully everybody who needs to account for this knows the change is coming.

Reply:
I'm not too worried other than our UI code breaking. Customers are unlikely to encounter such events as we will not emit webhooks and similar things for issueless events any time soon.

Reply:
Regarding the breaking change. Overall:
I did an analysis of the impact of groupID and group becoming nullable in several places:
Regarding the backend code, pretty much all the places I found that need the group from the event are unreachable for issueless events. Think about post processing, webhooks, group management code etc. We are not supposed to get there with issueless events.

For what concerns the UI, this is a lot up in the air for now. For what I know so far, the UI over issueless events is different, thank the event details, there are plenty of parts that are conceptually different that would prevent us from reusing the same one even if the group field was not None. 

I believe we should eventually have a totally separate data type for issueless events, but this is an open discussion as of now.

Reply:
The only UI that should bump into events without groupID is the event list page, and it new implementation. The new implementation is expecting issueless events for the most part. However, we're also missing UI components for transactions/spans which will cause different problems (all solvable though).